### PR TITLE
Mice have a small chance of chewing wires

### PR DESCRIPTION
**Pull request by Chronitonity**
from tg

Fight me on this its a 1% chance

### CODE REVIEW

**Comment by Foosili** [comment]
fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you fuck you

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -38,11 +39,14 @@
...
 
-/mob/living/simple_animal/mouse/death(gibbed)
+/mob/living/simple_animal/mouse/death(gibbed, toast)
 	if(!ckey)
 		..(1)
 		var/obj/item/trash/deadmouse/M = new(src.loc)
 		M.icon_state = icon_dead
+		if(toast)
+			M.color = "#3A3A3A"
+			M.desc = "It's toast."
```
Comment (danhibiki) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:49]: no its a mouse

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -55,6 +59,21 @@
...
 	..()
 
+/mob/living/simple_animal/mouse/handle_automated_action()
+	if(prob(chew_probability))
+		var/turf/simulated/floor/F = get_turf(src)
+		if(istype(F) && !F.intact)
+			var/obj/structure/cable/C = locate() in F
+			if(C && prob(15))
+				if(C.avail())
+					visible_message("<span class='warning'>[src] chews through the [C]. It's toast!</span>")
```
Comment (danhibiki) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:69]: its still a mouse

**Comment by Foosili** [comment]
http://www.nerdgranny.com/wp-content/uploads/2008/06/ascii-goatse.gif

**Comment by Kilmannan** [comment]
How many nice start on the station? Is it a 1% chance every time they cross a tile with a wire on it? How often do they move tiles?

Just wondering in case that 1% suddenly becomes a very regular occurrence with the full calculations.

**Comment by Weisyl** [comment]
fuckno

**Comment by doobiedoo23** [comment]
lmao no

**Comment by SplergeBucket** [comment]
oh hey i forget to comment
i, lou ping, did my best to deny this
-10000000 :-1: :100:

**Comment by danhibiki** [comment]
Yikes, looks like this is an overwhelming no, though I would have been interested in it

**Comment by blobbernaut** [comment]
rip power every fucking round

maaybe if you emag mice

**Comment by ghost** [comment]
i would be cool with this if the 1% chance could only happen once. you run the dice, lose, no more chances for the mice to cut wires

**Comment by Nebyab** [comment]
jesus fuck no

we had an argument about this weeks ago chron. this "feature" is awful, though at least you aren't trying to present it as messianic as that guy was

**Comment by Chronitonity** [comment]
@Foosili I didnt lose this you broke the terms of our bet

**Pull request closed without merging**